src/ott/solvers/nn/models.py

```diff
@@ -292,6 +286,7 @@ def create_train_state(
       rng: jax.random.PRNGKeyArray,
       optimizer: optax.OptState,
       input: Union[int, Tuple[int, ...]],
+      # TODO(michalk8): do not ignore or delete in code?
```

Comment:
@bamos are the `params` supposed to be ignored?

Reply:
`params` here are for initialization and shouldn't be ignored, they're passed into `NeuralTrainState.create`